tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_gluster_heal_info.py

```diff
@@ -21,17 +21,6 @@ def _parse_heal_info_stats(tree):
         brick_host = brick_name.split(":")[0]
         brick_path = brick_name.split(":")[1]
 
-        # If brick host is returned as an IP conver to FQDN
-        try:
-            from dns import resolver
-            from dns import reversename
-            from IPy import IP
-            IP(brick_host)
-            addr = reversename.from_address(brick_host)
-            brick_host = str(resolver.query(addr, "PTR")[0])[:-1]
```

Comment:
all details are stored in graphite using fqdn which we are passing, so while storing brick details also we have to replace brick hostname using fqdn and store it. otherwise, if the brick hostname is different from fqdn then it won't stored in a proper hierarchy.

Reply:
Ah yes agree. I missed somehow. this reverse naming would be required.